apps/game_client/assets/js/protobuf/messages_pb.js

```diff
@@ -22,6 +22,7 @@ var global =
     Function('return this')();
 
 goog.exportSymbol('proto.Attack', null, global);
+goog.exportSymbol('proto.AttackParameters', null, global);
```

Comment:
You have changes to game_client. Please restore them so that we don't mess with the other team's development process